src/main/resources/db/migration/V1_2__create_table_behandler_dialogmelding.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE BEHANDLER_DIALOGMELDING
+(
+    id            SERIAL PRIMARY KEY,
+    behandler_ref VARCHAR(50)  NOT NULL UNIQUE,
+    type          VARCHAR(30)  NOT NULL,
+    fnr           VARCHAR(11)  NOT NULL,
+    fornavn       VARCHAR(255) NOT NULL,
+    mellomnavn    VARCHAR(255),
+    etternavn     VARCHAR(255) NOT NULL,
+    partner_id    VARCHAR(50)  NOT NULL,
+    her_id        VARCHAR(50)  NOT NULL,
+    parent_her_id VARCHAR(50)  NOT NULL,
```

Comment:
Har alle behandlere (også andre enn fastlege) denne?

Reply:
Det tror jeg ikke.  Slik jeg har forstått det vil ha alltid ha minst 1 av disse to: orgnummer eller en herId(git man har funnet partner_id). Uavhengig av hvilken av disse vi har funnet, så får vi kanskje parentHerId.
Ikke 100% sikkert på dette, men oppsumert så tror jeg både parent_her_id og her_id må være nullable her om det skal kunne støtte ulike type behandlere.

Reply:
PS: jeg dobbeltsjekket dette med her_id vs parent_her_id i kontekst av partner_id nå.